src/PackageManagement.UI/InfiniteScrollListItemStyleSelector.cs

```diff
@@ -8,14 +8,36 @@ namespace NuGet.PackageManagement.UI
 {
     public class InfiniteScrollListItemStyleSelector : StyleSelector
     {
+        private static InfiniteScrollList InfiniteScrollList = new InfiniteScrollList();
+
+        private Style PackageItemStyle { get; set; }
+        private Style LoadingStatusIndicatorStyle { get; set; }
+
+        private void Init()
+        {
+            if (InfiniteScrollList != null && PackageItemStyle == null && LoadingStatusIndicatorStyle == null)
```

Comment:
InfiniteScrollList can never be null, because you never clear it. Instead of caching the scroll list, why won't you cache the styles? The right fix is to move the style out completely out of the infinite scroll list, and place it in a separate resource.